ceph-pr-commits/build/test_commits.py

```diff
@@ -14,6 +14,7 @@
 
 
 def run(command):
+    print "Running command: %s" % ' '.join(command)
```

Comment:
@alfredodeza do you want to print out the stdout.read() also just for debugging purpose?